### DISCUSSION TITLE
Automation Trigger: CRON - Implemented

### DISCUSSION THREAD

**Discussion by arladmin** [Budibase/budibase#1203] [Ideas]
A CRON option as a trigger would be a great addition.

Use-cases include running periodic audits, data-cleaning, report generation etc.

**Comment by joebudi** [comment]
This is another great suggestion @arladmin and one we will get round to eventually. To be honest, I feel this is pretty essential and will push at the next roadmap meeting.

**Comment by shogunpurple** [comment]
Addressed in: https://github.com/Budibase/budibase/issues/1221
